bamboo/set-bamboo-env-variables.sh

```diff
@@ -33,8 +35,8 @@ declare -a param_list=(
   "bamboo_SECRET_LAUNCHPAD_PASSPHRASE"
   "bamboo_SECRET_METRICS_ES_PASS"
   "bamboo_SECRET_NPM_TOKEN"
-  "bamboo_SECRET_ORCA_POSTGRES_USER_PASSWORD"
   "bamboo_SECRET_ORCA_DATABASE_APP_USER_PASSWORD"
+  "bamboo_SECRET_ORCA_POSTGRES_USER_PASSWORD"
   "bamboo_SECRET_PROVIDER_FTP_PORT"
   "bamboo_SECRET_PROVIDER_HTTP_PORT"
   "bamboo_SECRET_RDS_ADMIN_ACCESS_SECRET_ARN"
```

Comment:
alphabetizing 🤩

Reply:
That's courtesy of @flamingbear  pointing it out :)